### DISCUSSION TITLE
timeshift

### DISCUSSION THREAD

**Discussion by rgparkins** [blackcowmoo/Grafana-Google-Analytics-DataSource#89] [Q&A]
Hi all

Just as a starter for 10 does anyone know if you can display, for example, data from today and then data from same time last week on the same visualisation. I have done this before with 2 queries (a long time ago) but there is talk on the web about this only being supported by specific data sources, does this datasource support this?

Many thanks

Richard

**Comment by lcc3108** [comment]
No separate logic has been created for the time shift.
I will answer you after the test.

**Reply by rgparkins** [reply]
yeah its a strange one as I thought it would be part of the grafana UI as opposed to the data source itself as the time series selected is controlled by grafana UI for all panels I believe

**Reply by lcc3108** [reply]
Timeshifts can delay time for a grafana panel (there can be multiple queries in one panel, each of which cannot be set)

If you want to time-shift each query in one panel, the plug-in must support it.

It is not currently supported.
